manifests/base/kustomization.yaml

```diff
@@ -12,3 +12,7 @@ resources:
 - deployment.yaml
 - routes.yaml
 - service.yaml
+images:
+  - name: odh-dashboard
+    newName: quay.io/opendatahub/odh-dashboard
+    newTag: v1.0
```

Comment:
Despite the fact that the tag will change once we have a new image, right now it seems that the latest version was v1.0.0.1, not sure if there are some significant changes between versions.